### ISSUE TITLE
Refactor newsletter | news | newsletter_archive

### ISSUE THREAD

**Issue by jvolker** [technologiestiftung/citylab-microsite#64]
`Newsletter`, `news` and `newsletter_archive` are used interchangeably but they are not the same. This should be fixed.

**Comment by Esshahn** [comment]
still relevant after your latest change? I saw some renaming in the PR.
If not relevant anymore, please close.

**Comment by jvolker** [comment]
Yes, still relevant for this codebase. But not relevant if we start from scratch.
